store/customPreset.js

```diff
@@ -0,0 +1,86 @@
+import { groupRepeatedUnits, createRepetition } from '../utils/helpers'
+
+export const state = () => ({
+  columns: 4,
+  rows: 2,
+  colArr: [],
+  rowArr: [],
```

Comment:
what is `childarea` and `preview` area?

Reply:
I've renamed them to ***zones** and ***previewZone***